protocol/requests/input_connection_string_for_bootstrapping.go

```diff
@@ -0,0 +1,6 @@
+package requests
+
+type InputConnectionStringForBootstrapping struct {
+	ConnectionString string `json:"connectionString"`
+	ConfigJSON       string `json:"configJSON"`
```

Comment:
Specific type?

Reply:
I think it's better to fix right away. Otherwise it will stay like this forever. We would have to implement a V3 endpoint to change it 😄

Reply:
Maybe merge all other endpoints first, and work with this one as a separate PR?